packages/back-end/src/services/savedGroups.ts

```diff
@@ -0,0 +1,20 @@
+import { OrganizationInterface } from "../../types/organization";
```

Comment:
@jdorn I took a pass at building the webHook update call when a savedGroup's values are updated - I'm not super confident in this code, but I'm essentially just passing in all environments & all projects and letting the `queueWebhooks` decide if a webhook needs to get called. This feels like it could get expensive, so I added a check in the controller to only call this is the savedGroup `values` changes.

I'm also not entirely sure if the final argument of the `queueWebhooks` call should be `true` or `false`.